src/Specifications.php

```diff
@@ -29,6 +34,40 @@ class Specifications
      */
     public const VEHICLE_TYPE_DEFAULT = 'ID_TYPE_CAR';
 
+    /**
+     * Get current package version.
+     *
+     * @param bool $without_hash
+     *
+     * @return string
+     */
+    public static function version(bool $without_hash = true): string
+    {
+        $version = Versions::getVersion(self::AVTOCOD_SPECS_PACKAGE_NAME);
+
+        if ($without_hash === true && \is_int($delimiter_position = \mb_strpos($version, '@'))) {
+            return \mb_substr($version, 0, (int) $delimiter_position);
+        }
+
+        return $version;
+    }
+
+    /**
+     * Get the specifications root directory path.
+     *
+     * @param string|null $additional_path
+     *
+     * @return string
+     */
+    public static function getRootDirectoryPath(string $additional_path = null): string
+    {
+        $root = \ComposerLocator::getPath(static::AVTOCOD_SPECS_PACKAGE_NAME);
```

Comment:
Or use reflection + `. self::AVTOCOD_SPECS_PACKAGE_NAME`?

Reply:
Made in [db8f1c0](https://github.com/avtocod/specs-php/pull/1/commits/db8f1c0c2f6ea09f5469543d43c5b98dcc98a2b2)